Dockerfile

```diff
@@ -1,4 +1,10 @@
-FROM ubuntu:16.04
+FROM openjdk:8-jdk
+
+ARG VERSION=3.7
```

Comment:
will do (technically it's the Jenkins [Remoting](https://github.com/jenkinsci/remoting) Module version; will update accordingly)